crates/e2e/tests/e2e/limit_orders.rs

```diff
@@ -446,3 +457,100 @@ async fn too_many_limit_orders_test(web3: Web3) {
     assert_eq!(status, 400);
     assert!(body.contains("TooManyLimitOrders"));
 }
+
+async fn forked_single_limit_order_test(web3: Web3) {
+    let mut onchain = OnchainComponents::at(web3.clone()).await;
+    let forked_node_api = web3.api::<ForkedNodeApi<_>>();
+
+    let auth_manager = onchain
+        .contracts()
+        .gp_authenticator
+        .manager()
+        .call()
+        .await
+        .unwrap();
+
+    forked_node_api.impersonate(&auth_manager).await.unwrap();
+
+    let [solver] = onchain
+        .make_solvers_forked(to_wei(1), ethcontract::Account::Local(auth_manager, None))
+        .await;
```

Comment:
building the `forked_node_api` and impersonating the `authenticator` could be folded into `make_solvers_forked()`, right?
Is there ever a reason to use different addresses for this?